<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateScheduleModal</class>
 <widget class="QWidget" name="UpdateScheduleModal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-weight: 600; font-size: 10pt; color: black;</string>
     </property>
     <property name="text">
      <string>Расписание</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="formLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="inputGroup_8">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="trainLabel">
             <property name="text">
              <string>Поезд</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="trainComboBox"/>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="inputGroup_9">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="arrivalDateLabel">
             <property name="text">
              <string>Время прибытия</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="arrDateInput"/>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="inputGroup_7">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="routeLabel">
             <property name="text">
              <string>Маршрут</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="routeComboBox"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="inputGroup_10">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="departureDateLabel">
             <property name="text">
              <string>Время отправления</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="depDateInput"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Сохранить изменения</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
